### PR TITLE
Fix/change create order response

### PR DESCRIPTION
**Pull request by aligent-skumar**
Return the order object in the create order response.
Changes made in src/BigCommerceLegacyApi/Api/Orders/OrdersApi.php (Line 28)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,20 +79,20 @@ try {
 
 There are three components to the library:
 
-- [BigCommerce/Api](./src/BigCommerce/Api) - which represent the API endpoints and tries to mimic the layout of the
```
Comment (jswift) [inline README.md:82]: why did these change?

File: composer.json [hunk 2]
```diff
@@ -27,12 +27,12 @@
...
     "require-dev": {
-        "phpunit/phpunit": "^9",
-        "squizlabs/php_codesniffer": "3.*"
+        "../../vendor/phpunit/phpunit": "^9",
+        "../../vendor/squizlabs/php_codesniffer": "3.*"
     },
     "scripts": {
         "test": "./vendor/bin/phpunit tests",
-        "check-style": "./vendor/bin/phpcs --standard=PSR12 -s ./src ./tests",
-        "fix-style": "./vendor/bin/phpcbf --standard=PSR12 ./src ./tests"
```
Comment (jswift) [inline composer.json:36]: I'm confused why all the relative paths are being changed

File: composer.json [hunk 3]
```diff
@@ -27,12 +27,12 @@
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "^9",
-        "squizlabs/php_codesniffer": "3.*"
+        "../../vendor/phpunit/phpunit": "^9",
```
Comment (jswift) [inline composer.json:30]: this isn't good, will break

**Review by jswift — CHANGES_REQUESTED** [review]
The actual change to the code will work, but why are there all these other changes? It should only be a one line change

**Comment by aligent-skumar** [comment]
I think I moved the repository locally. I'll create PR again.

**Pull request closed without merging**